bftengine/src/bftengine/ReplicaImp.hpp

```diff
@@ -12,7 +12,6 @@
 #pragma once
 
 #include <string>
-#include "Replica.hpp"
```

Comment:
Why did you remove `Replica.hpp` - `IRequestsHandler` is used in here and it is defined in the aforementioned header.

Reply:
it is included with RequestHandler.h